src/index.html

```diff
@@ -8,6 +8,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="card img"
```

Comment:
Check the [checklist №13](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md), alt needs to contain information of the image content.